WordPress/gutenberg
# Fully remove Navigation Editor screen and associated `edit-navigation` package

## What?


Removes the Navigation Editor screen from codebase and removes `@wordpress/edit-navigation` screen.



Closes https://github.com/WordPress/gutenberg/issues/43620

## Why?


For some time now this screen has been:
- removed from the Gutenberg UI
- deprecated 

As there are no plans to continue working on this concept it's time to remove it and the associated package.

Note that many learnings were taken from the work on this package which have since been applied to other projects surrounding Navigation. But...it's time to let it go 😢 


## How?


Removes all file references.

## Testing Instructions





- Check all editors work as expected
- Check for PHP errors
- Check codebase for references to the package or files.
- Check I removed the package correctly.

### Testing Instructions for Keyboard


## Screenshots or screencast
